### DISCUSSION TITLE
[Feature Request] Support InternLM

### DISCUSSION THREAD

**Discussion by vansinhu** [KudoAI/chatgpt.js#244] [Ideas]
Dear chatgpt.js developer,

Greetings! I am vansinhu, a community developer and volunteer at InternLM. Your work has been immensely beneficial to me, and I believe it can be effectively utilized in InternLM as well. Welcome to add Discord https://discord.gg/gF9ezcmtM3 . I hope to get in touch with you.

Best regards,
vansinhu

**Comment by vansin** [comment]
> Hey @vansinhu thanks for reaching out! May I ask how chatgpt.js has immensely benefited you?

help me can easily run the chatgpt ui as long as I know a little nodejs

**Comment by vansin** [comment]
InternLM is a Large Language Model like LLama, chatgpt.js can help InternLM to create a UI

**Comment by vansinhu** [comment]
> @vansin i see, what is internLM and what would example of adding support for it be?

for example i add a button so users can get ideas, this is really good.
